portal/common/lib/rpc_selector.ts

```diff
@@ -80,7 +92,10 @@ class RPCSelector implements RPCSelectorInterface {
         }
 
         const timeoutPromise = new Promise<never>((_, reject) =>
-            setTimeout(() => reject(new Error("Request timed out")), RPC_REQUEST_TIMEOUT_MS),
+            setTimeout(() => reject(
+                new Error("Request timed out")),
+                Number(process.env.RPC_REQUEST_TIMEOUT_MS) ?? 7000
```

Comment:
Question: why `7000`? Why should `process.env.RPC_REQUEST_TIMEOUT_MS` not be present?
Suggestion: make this a const at least.

Reply:
That's a remnant actually. I think we initially decided it to be this big out of convenience. 
Not sure what the best value would be for this, but we can try different configurations to see what fits best.